client/js/test/table-view.test.js

```diff
@@ -28,6 +28,24 @@ describe('table-view', () => {
   });
   });	
 
```

Comment:
Add a few more tests that test the functionality of your new feature. Especially test against non-happy paths and edge cases. For instance, “it correctly generates sum when the column includes non-number inputs”.